launcher/container_runner.go

```diff
@@ -532,25 +540,53 @@ func initImage(ctx context.Context, cdClient *containerd.Client, launchSpec spec
 	return image, nil
 }
 
-func getImageLabels(ctx context.Context, image containerd.Image) (map[string]string, error) {
-	// TODO(jiankun): Switch to containerd's WithImageConfigLabels()
+// openPorts writes firewall rules to accept all traffic into that port and protocol using iptables.
+func openPorts(ports map[string]struct{}) error {
+	for k := range ports {
+		portAndProtocol := strings.Split(k, "/")
```

Comment:
where is this documented? I couldn't find it inhttps://pkg.go.dev/github.com/opencontainers/image-spec/specs-go/v1#ImageConfig. and https://docs.docker.com/config/containers/container-networking/ seems to default to tcp when the protocol is not specified